server/routes/index.js

```diff
@@ -1,8 +1,23 @@
 import { Router } from 'express';
+import passport from 'passport';
 import userAuth from './api/auth/userAuth';
+import Facebook from '../controllers/auth/strategies/facebookStrategy';
 
-const router = Router();
 
+const router = Router();
 router.use('/api/v1', userAuth);
+// router.use('/api', api);
+router.get('/api/v1/facebook',
```

Comment:
Nice one Daniel, Instead of having this route `api/v1/facebook` here, I think it will be better if it is in another file. This file should just be for bringing all our routes together from our different files.

Reply:
Okay wprking on it